jetpack/plan-provisioning-direct-api.md

```diff
@@ -100,13 +100,20 @@ Plans can be provisioned by making a request using your partner token from the s
 
 ### Request Parameters (/provision)
 
-- __local_user__:     The username, ID or email on the local website (not the WordPress.com username) that should own the plan. The corresponding user _must_ already exist.
-- __siteurl__:        The URL where the WordPress core files reside.
-- __plan__:           A slug representing which plan to provision. One of `free`, `personal`, `premium`, or `professional`.
-- __force_register__: (optional) A true/false value indicating whether to re-register a site even if we already have tokens for it. Useful for sites that have gotten into a bad state.
-- __force_connect__:  (optional) A true/false value indicating whether to re-connect a user even if we already have tokens for them. Useful for sites that have gotten into a bad state.
-- __onboarding__:     (optional) If true, put the user through our onboarding wizard for new sites.
-- __wpcom_user_id__:  (optional) For certain keys, enables auto-connecting a WordPress.com user to the site non-interactively.
+- __local_user__:      The username, ID or email on the local website (not the WordPress.com username) that should own the plan. The corresponding user _must_ already exist.
+- __siteurl__:         The URL where the WordPress core files reside.
+- __plan__:            A slug representing which plan to provision. One of `free`, `personal`, `premium`, or `professional`.
+- __force_register__:  (optional) A true/false value indicating whether to re-register a site even if we already have tokens for it. Useful for sites that have gotten into a bad state.
+- __force_connect__:   (optional) A true/false value indicating whether to re-connect a user even if we already have tokens for them. Useful for sites that have gotten into a bad state.
+- __onboarding__:      (optional) If true, put the user through our onboarding wizard for new sites.
+- __wpcom_user_id__:   (optional) For certain keys, enables auto-connecting a WordPress.com user to the site non-interactively.
+- __ssh_user__:        (optional) Set SSH user.
+- __ssh_pass__:        (optional) Set SSH password.
```

Comment:
It may be worth mentioning that this field, or the next, is required if `ssh_user` is set.

Reply:
Scroll down 5 lines 😛